server/channels/store/sqlstore/remote_cluster_store.go

```diff
@@ -22,6 +22,25 @@ func newSqlRemoteClusterStore(sqlStore *SqlStore) store.RemoteClusterStore {
 	return &sqlRemoteClusterStore{sqlStore}
 }
 
+func remoteClusterFields(prefix string) []string {
+	if prefix != "" && !strings.HasSuffix(prefix, ".") {
+		prefix = prefix + "."
+	}
```

Comment:
0/5 but this seems like we are doing it to be extra-safe, when we could have just passed "rc." or "sc." from the call site itself.

Reply:
Yes, it's just a pattern I use in a bunch of places so there is one place where all the columns of a table are listed, and I don't have to remember if the period is needed or not.